pkg/webui/components/breadcrumbs/breadcrumb/breadcrumb.styl

```diff
@@ -31,6 +31,11 @@
     text-decoration: underline
     color: $tc-deep-gray
 
+.last
+  one-liner()
+  nudge('up')
```

Comment:
```suggestion
```
Without this, it is positioned correctly.

Reply:
@kschiffer I added `nudge('up')` to fix the issue with alignment when there is no icons. if I remove it the last breadcrumb is not aligned again.

Reply:
I see. I was able to fix the issue by applying a `min-height` to the breadcrumb `<a>` and `<span> // for the last one` and using `display: flex` for them. No more nudging needed then.

Reply:
okay great will do that

Reply:
I think adding `min-height` alone fixes the issue without `display: flex`